src/ui/view/dataverse/component/filters/selectionFilter/selectionFilter.tsx

```diff
@@ -0,0 +1,144 @@
+import type { FC } from 'react'
+import { useMemo, useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Collapsible } from '@/ui/component/collapsible/collapsible'
+import { SearchBar } from '@/ui/component/searchbar/searchbar'
+import { isSubstringOf } from '@/util/util'
+import { Icon } from '@/ui/component/icon/icon'
+import { DynamicCheckbox } from '@/ui/view/dataverse/component/dynamicCheckbox/dynamicCheckbox'
+import './selectionFilter.scss'
+
+export type SelectionOption = {
+  selected: boolean
+  value: string
+  name: string
+  disabled: boolean
+}
+
+type SelectionItemType = 'checkbox'
+
+const NoResultsFound: FC = () => {
+  const { t } = useTranslation('common')
+  return (
+    <div className="okp4-dataverse-portal-selection-filter-no-results">
+      <Icon name="large-magnifier-with-cross" />
+      <span>{t('noResultsFound')}</span>
+    </div>
+  )
+}
+
+type FilterNameProps = {
+  children: string
+}
+export const FilterName: FC<FilterNameProps> = ({ children }) => (
+  <h3 className="okp4-dataverse-portal-dataverse-filter-name">{children}</h3>
+)
+
+type SelectionFilterProps = {
+  filterName: string
+  filterValues: string[]
```

Comment:
I can see that you sometimes uses `options` to describe `filterValues`, you could have named `filterName`: `label` and `filterValues`: `options`. Not really important, you do as you prefer!

Reply:
Yes `label` might be an appropriate naming too. The`filterName` naming was chosen to match the `name` props of the `SelectionOption`.
The Filter options is more like a `SelectionOption`:
```tsx
type SelectionOption = {
    selected: boolean;
    value: string;
    name: string;
    disabled: boolean;
}
```